packages/auth/src/AuthProvider.tsx

```diff
@@ -115,8 +115,11 @@ export class AuthProvider extends React.Component<
   }
 
   getCurrentUser = async (): Promise<Record<string, unknown>> => {
+    if (!window) {
+      return {}
+    }
```

Comment:
Could we be explicit here and check for `process.env.__REDWOOD_PRERENDERING`?

Reply:
I've removed this bit, just opting to use global.fetch instead